op-batcher/compressor/ratio_compressor_test.go

```diff
@@ -1,113 +1,54 @@
 package compressor_test
 
 import (
+	"fmt"
 	"math"
 	"testing"
 
 	"github.com/ethereum-optimism/optimism/op-batcher/compressor"
 	"github.com/stretchr/testify/require"
 )
 
-// TestInputThreshold tests the [ChannelConfig.InputThreshold]
-// function using a table-driven testing approach.
-func TestInputThreshold(t *testing.T) {
-	type testInput struct {
-		TargetFrameSize  uint64
-		TargetNumFrames  int
-		ApproxComprRatio float64
-	}
-
-	// Construct test cases that test the boundary conditions
+func TestChannelConfig_InputThreshold(t *testing.T) {
 	tests := []struct {
-		input     testInput
-		assertion func(uint64)
+		targetOutputSize  uint64
+		approxComprRatio  float64
+		expInputThreshold uint64
+		assertion         func(uint64) // optional, for more complex assertion
 	}{
 		{
-			input: testInput{
-				TargetFrameSize:  1,
-				TargetNumFrames:  1,
-				ApproxComprRatio: 0.4,
-			},
-			assertion: func(output uint64) {
-				require.Equal(t, uint64(2), output)
-			},
+			targetOutputSize:  1,
+			approxComprRatio:  0.4,
+			expInputThreshold: 2,
 		},
 		{
-			input: testInput{
-				TargetFrameSize:  1,
-				TargetNumFrames:  100000,
-				ApproxComprRatio: 0.4,
-			},
-			assertion: func(output uint64) {
-				require.Equal(t, uint64(250_000), output)
-			},
+			targetOutputSize:  1,
+			approxComprRatio:  1,
+			expInputThreshold: 1,
```

Comment:
I removed a couple of cases which became irrelevant because I removed the target num frames. otherwise all test cases map to previous ones, the diff just shifted some blocks around. the only thing this test tests is that `targetOutputSize / approxComprRatio = expInputThreshold`